corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py

```diff
@@ -171,15 +175,13 @@ def ensure_plans(dry_run, verbose, for_tests, apps):
                     if hasattr(SoftwarePlanVersion, 'product_rates'):
                         software_plan_version.save()
 
-                    for product_rate in product_rates:
-                        product_rate.save()
-                        if hasattr(SoftwarePlanVersion, 'product_rates'):
-                            software_plan_version.product_rates.add(product_rate)
-                        elif hasattr(SoftwarePlanVersion, 'product_rate'):
-                            assert len(product_rates) == 1
-                            software_plan_version.product_rate = product_rate
-                        else:
-                            raise AccountingError('SoftwarePlanVersion does not have product_rate or product_rates field')
+                    product_rate.save()
+                    if hasattr(SoftwarePlanVersion, 'product_rates'):
```

Comment:
This is necessary because this code is run in a migration before https://github.com/dimagi/commcare-hq/pull/9781 (see ```0001_initial.py```).  This is to satisfy backwards compatibility